SQL/Release_patches/21.0_To_22.0_upgrade.sql

```diff
@@ -0,0 +1,107 @@
+  CONSTRAINT `FK_user_project_rel_ProjectID` FOREIGN KEY (`ProjectID`) REFERENCES `Project` (`ProjectID`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- To maintain status quo, add all projects to all users since users had access to all data before projects were added.
+INSERT IGNORE INTO user_project_rel
+SELECT ID,ProjectID FROM users JOIN Project;
+
+SELECT 'Running: SQL/Archive/22.0/2019-10-01_Rename-media-column.sql';
+
+ALTER TABLE media CHANGE date_uploaded last_modified timestamp;
+
+SELECT 'Running: SQL/Archive/22.0/2019-10-05-Add_alias_to_projects.sql';
+
+ALTER TABLE `Project` ADD COLUMN `Alias` char(4) DEFAULT NULL AFTER `Name`;
+UPDATE Project SET Alias=UPPER(LEFT(Name,4)) WHERE Alias IS NULL;
+ALTER TABLE `Project` CHANGE COLUMN `Alias` `Alias` char(4) NOT NULL;
+
+SELECT 'Running: SQL/Archive/22.0/2019-11-01-Add_data_release_permissions.sql';
+
+INSERT INTO permissions (code,description,categoryID) VALUES 
+('data_release_view','Data Release: View releases',(SELECT ID FROM permissions_category WHERE Description='Permission'));
+
+INSERT INTO LorisMenuPermissions (MenuID, PermID)
+    SELECT m.ID, p.PermID FROM permissions p CROSS JOIN LorisMenu m WHERE p.code='data_release_view' AND m.Label='Data Release';
+
+INSERT IGNORE INTO user_perm_rel
+SELECT upr.userID, p.permID FROM user_perm_rel upr JOIN permissions p WHERE upr.permID=1 AND p.code IN ('data_release_view');
+
+SELECT 'Running: SQL/Archive/22.0/2019-11-12-Rename_modules_QC_and_DTH.sql';
+
+-- rename modules and change order number according to the new parent menu they belong to
+SELECT MAX(OrderNumber)+1 INTO @ordernumber FROM LorisMenu WHERE Parent=(SELECT ID FROM LorisMenu WHERE Label='Clinical');
+SELECT ID INTO @parentid FROM LorisMenu WHERE Label='Clinical';
+
+UPDATE LorisMenu SET Label='Behavioural Quality Control', Link='behavioural_qc/', Parent=@parentid, OrderNumber=@ordernumber  WHERE Link='data_team_helper/';
+
+SELECT MAX(OrderNumber)+1 INTO @ordernumber FROM LorisMenu WHERE Parent=(SELECT ID FROM LorisMenu WHERE Label='Imaging');
+SELECT ID INTO @parentid FROM LorisMenu WHERE Label='Imaging';
+UPDATE LorisMenu SET Label='Imaging Quality Control', Link='imaging_qc/', Parent=@parentid, OrderNumber=@ordernumber WHERE Link='quality_control/';
+
+-- change location of Genomic Browser to place right after imaging tab
+SELECT OrderNumber INTO @ordernumber FROM LorisMenu WHERE Label='Imaging';
+UPDATE LorisMenu SET OrderNumber=(OrderNumber+1) WHERE Parent IS NULL AND OrderNumber>@ordernumber;
+
+INSERT INTO LorisMenu (Label, OrderNumber)
+SELECT 'Genomics', OrderNumber+1 FROM LorisMenu WHERE Label='Imaging';
+
+SELECT ID INTO @parentid FROM LorisMenu WHERE Label='Genomics';
+UPDATE LorisMenu SET Parent=@parentid WHERE Link='genomic_browser/';
+
+UPDATE permissions SET Code='quality_control', description='Quality Control access' WHERE code='data_team_helper';
```

Comment:
```suggestion
UPDATE permissions SET Code='quality_control', description='Quality Control access' WHERE code='data_team_helper';
```

Should we make a ticket to create a unique permission for this?

Reply:
Also maybe the description here should say Behavioural/Imaging QC depending on which permission it actually is. It's not clear from here.

Reply:
it's both.

the reason is that whoever coded the imaging_qc module used the `data_team_helper` permission to control access to it. I renamed the permission to `quality_control` to be more generic and yes I do think they shou;d have each their own permissions it was just out of scope to do it  in that PR.